report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html

```diff
@@ -35,7 +35,7 @@ <h4 class="toolbar-link-header ellipsis">
         data-content="Rerun current URL"
         data-trigger="hover"
         data-container="body"
-        data-ng-click="toolbarBottom.rerunURL(toolbarBottom.model.url)">
+        data-ng-click="toolbarBottom.rerunURL(toolbarBottom.model.name)">
```

Comment:
Will it also work when URL doesn't have a "name" attribute in XML?

Reply:
Yes, because when user doesn't set "Name" attribute, then URL is set as "Name"